float64[] pos
---
bool ret
